core/primitives/src/runtime/parameter_table.rs

```diff
@@ -139,22 +218,27 @@ fn parse_parameter_txt_value(value: &str) -> Result<serde_json::Value, InvalidCo
         return Ok(serde_json::Value::Null);
     }
     if value.chars().all(|c| c.is_numeric() || c == '_') {
```

Comment:
Does json allow all unicode numeric codepoints? May want to check for `is_ascii_digit` (in which case you can also iterate through bytes instead of characters) instead.

Reply:
I didn't even think about this. But I looked it up, [JSON grammar](https://datatracker.ietf.org/doc/html/rfc7159#section-6) for number values is very strict and has no room for Unicode interpretation. So I changed it as you suggested.